packages/replayio/src/commands/record.ts

```diff
@@ -56,20 +54,18 @@ async function record(url: string = "about:blank") {
     }
   }
 
-  const recordingsAfter = await getRecordings();
+  const recordingsAfter = await getRecordings(processGroupId);
```

Comment:
oooh, i thought before those groups were generated by Chromium and not by us. So this is more like a recording session id or smth like that. It makes sense that we want to get rid of this for now 👍

Reply:
Yup, yup.